streamer/parser/metadata.md

```diff
@@ -111,52 +105,56 @@ The contributor object may also contain a `sortAs` string, used to sort the cont
 When parsing an EPUB, we need to establish:
 
 * the key of the contributor;
-* the name of this contributor;
-* the alternate forms for this name;
+* a language map for the name of this contributor;
 * the string used to sort the name of the contributor.
 
 ### EPUB 2.x
 
 The following mapping should be used to determine the key of the contributor’s object: 
 
-| element        | opf:role               | key         |
-|----------------|------------------------|-------------|
-| dc:creator     | aut                    | author      |
-| dc:contributor | trl                    | translator  |
-| dc:contributor | est                    | editor      |
-| dc:contributor | ill                    | illustrator |
-| dc:contributor | art                    | artist      |
-| dc:contributor | clr                    | colorist    |
-| dc:contributor | nrt                    | narrator    |
-| dc:contributor | \<empty\> or \<other\> | contributor |
+| element        | opf:role                        | key         |
+|----------------|---------------------------------|-------------|
+| dc:creator     | aut or \<empty\> or \<unknown\> | author      |
+| dc:publisher   | pbl or \<empty\> or \<unknown\> | publisher   |
+| dc:contributor | trl                             | translator  |
+| dc:contributor | edt                             | editor      |
+| dc:contributor | ill                             | illustrator |
+| dc:contributor | art                             | artist      |
+| dc:contributor | clr                             | colorist    |
+| dc:contributor | nrt                             | narrator    |
+| dc:contributor | \<empty\> or \<unknown\>        | contributor |
 
 Where `opf:role` is the value of the attribute of the `<dc:element>`.
 
-The `name` of the contributor is the value of the element.
+In case of conflict, `opf:role` overrides the XML element used. For example, map a `<dc:publisher>` or `<dc:contributor>` element whose value for `opf:role` is `aut` to an author.
```

Comment:
Alright, so I now know why that made me pause yesterday. 

I would rephrase the “case of conflict” into a case of extension/augmentation since:

> The [DC11] contributor element is used to represent the name of a person, organization, etc. that played a secondary role in the creation of the content of an EPUB Publication. ([source](https://www.w3.org/publishing/epub3/epub-packages.html#sec-opf-dccontributor))

So in theory, `<dc:contributor> + aut` should be something of a secondary author. But maybe I’m wrong about that, please feel free to chime in.

Then `role` does not apply to `<dc:publisher>`, cf. 

> Extends: contributor, creator

in [the EPUB 3.2 spec](https://www.w3.org/publishing/epub3/epub-packages.html#sec-role). Same in [the EPUB 2.0.1 spec](http://idpf.org/epub/20/spec/OPF_2.0.1_draft.htm#Section1.3.3): “the OPF role attribute added to the Dublin Core `creator` and `contributor` elements”.

Reply:
> So in theory, <dc:contributor> + aut should be something of a secondary author. But maybe I’m wrong about that, please feel free to chime in.

If it is so, maybe our rule is not founded and we should map <dc:contributor> + aut  to a contributor...

> Then role does not apply to <dc:publisher>, cf.

You're right, indeed.

Reply:
Given some producers may put several authors in a single `<dc:creator>` so that both/all names are displayed in the UI, I’d imagine that if you can find `<dc:contributor> + aut`, there’s an underlying intent to make it secondary, and it’s a nuance to probably take into account.

But then I guess it could be useful to check if others have any thoughts? Now that we are fine-tuning that, I’m wondering whether `creator` and `contributor` have been handled well enough so far in this doc. I’m not sure about that, since these `roles` may apply to both.

Reply:
Opened an issue here: https://github.com/readium/architecture/issues/126

Reply:
I agree that `role` should not be applied to `<dc:publisher>`.

`<dc:creator>` and `<dc:contributor>` are unfortunately very vague, `creator` is not as specific as `author` for instance.
I think these two elements are vague enough to ignore the element and override them with `role`.

Reply:
Given the table has been updated, remove.